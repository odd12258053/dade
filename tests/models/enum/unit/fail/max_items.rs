use dade::model;
#[model]
enum TestModel {
    #[field(max_items = 2)]
    Value
}
fn main() {}
