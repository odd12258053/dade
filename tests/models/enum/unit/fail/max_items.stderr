error: Only support alias term on the unit field.
 --> tests/models/enum/unit/fail/max_items.rs:5:5
  |
5 |     Value
  |     ^^^^^
